packages/grafana-llm-app/pkg/mcp/live_server.go

```diff
@@ -148,24 +149,44 @@ type liveSession struct {
 	grafanaIdToken string
 }
 
+// tokenTimeoutSeconds is the expiration time for the Grafana Live session token.
+var tokenTimeoutSeconds = int((time.Minute * 30).Seconds())
+
+// tokenRefreshInterval is the time between token refreshes.
+var tokenRefreshInterval = time.Duration(tokenTimeoutSeconds) - time.Minute
```

Comment:
Doesn't this need to be something like the following?
```suggestion
var tokenRefreshInterval = time.Duration(tokenTimeoutSeconds) * time.Second - time.Minute
```